components/button/demo/button-group.md

```diff
@@ -1,69 +1,78 @@
 ---
 order: 5
-title: 按钮组合
+title:
+  zh-CN: 按钮组合
+  en-US: Button Group
 ---
 
+## zh-CN
+
 可以将多个 `Button` 放入 `Button.Group` 的容器中。
 
 通过设置 `size` 为 `large` `small` 分别把按钮组合设为大、小尺寸。若不设置 `size`，则尺寸为中。
 
+## en-US
+
+If we put several `Button` into `Button.Group`, they will become a button group.
+
+Set `size` as `large` or `small` or nothing, we will get large/small/defualt button group respectively.
```

Comment:
Buttons can be grouped by placing multiple `Button` components into a `Button.Group`.

The `size` can be set to `large`, `small` or left unset resulting in a default size.